podcasts/en.lproj/Localizable.strings

```diff
@@ -4845,3 +4845,33 @@
 
 /* Settings to control sending of pocket casts offers notifications */
 "notifications_pocket_cast_offers" = "Pocket Casts Offers";
+
+/* Encourage Account Creation: modal title */
+"eac_informational_view_modal_title" = "We noticed you’re not logged in";
+
+/* Encourage Account Creation: modal description */
+"eac_informational_view_modal_description" = "Create an account or log in to enjoy\nPocket Casts to the fullest.";
+
+/* Encourage Account Creation: Get Started button */
+"eac_informational_view_modal_get_started_button" = "Get Started";
+
+/* Encourage Account Creation: card title for sync feature */
+"eac_informational_card_sync_title" = "Sync across devices";
+
+/* Encourage Account Creation: card title for backups feature */
+"eac_informational_card_backups_title" = "Reliable backups";
+
+/* Encourage Account Creation: card title for recommendations feature */
+"eac_informational_card_recommendation_title" = "Personalized recommendations";
+
+/* Encourage Account Creation: card description for sync feature */
+"eac_informational_card_sync_description" = "Sync your progress, and shows across all your devices.";
+
+/* Encourage Account Creation: card description for backups feature */
+"eac_informational_card_backups_description" = "Your library and preferences are securely saved.";
+
+/* Encourage Account Creation: card description for recommendations feature */
+"eac_informational_card_recommendation_description" = "Get tailored podcast suggestions based on your listening habits.";
+
+/* Login button */
+"login" = "Login";
```

Comment:
There is an existing entry called `account_login` that also has the login string.